src/components/Editor.tsx

```diff
@@ -69,7 +84,7 @@ export const Editor = ({
     }
 
     return getLocalStorageKey(
-      !!templateMetadata.isThemeMode,
+      templateMetadata.isThemeMode,
```

Comment:
Why is theme mode relevant to the non-theme key? We have a separate buildThemeKey func now